plasma_cash/utils/merkle/sparse_merkle_tree.py

```diff
@@ -46,8 +47,30 @@ def create_tree(self, ordered_leaves, depth, default_nodes):
                     else:
                         next_level[index // 2] = sha3(default_nodes[level] + value)
                 prev_index = index
+            tree.append(tree_level)
             tree_level = next_level
         return tree_level[0]
 
+    def create_merkle_proof(self, uid):
```

Comment:
minor: might worth document that the proof is concation of hashes from leaf to root